test/Core.Test/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommandTests.cs

```diff
@@ -377,57 +307,40 @@ public class UpdateSecretsManagerSubscriptionCommandTests
             Id = organizationId,
             UseSecretsManager = true,
             SmSeats = 10,
-            SmServiceAccounts = 200,
             MaxAutoscaleSmSeats = 20,
+            SmServiceAccounts = 100,
             MaxAutoscaleSmServiceAccounts = 350,
             PlanType = planType,
             GatewayCustomerId = "1",
             GatewaySubscriptionId = "2"
         };
 
         var plan = StaticStore.SecretManagerPlans.FirstOrDefault(x => x.Type == organization.PlanType);
-        var organizationUpdate = new SecretsManagerSubscriptionUpdate
-        {
-            OrganizationId = organizationId,
-            MaxAutoscaleSmSeats = 15,
-            SmSeatsAdjustment = 5,
-            MaxAutoscaleSmServiceAccounts = 300,
-            SmServiceAccountsAdjustment = 100,
-            SmSeats = organization.SmSeats.GetValueOrDefault() + 5,
-            SmSeatsExcludingBase = (organization.SmSeats.GetValueOrDefault() + 5) - plan.BaseSeats,
-            SmServiceAccounts = organization.SmServiceAccounts.GetValueOrDefault() + 100,
-            SmServiceAccountsExcludingBase = (organization.SmServiceAccounts.GetValueOrDefault() + 100) - (int)plan.BaseServiceAccount,
-            MaxAutoscaleSmSeatsChanged = 15 != organization.MaxAutoscaleSeats.GetValueOrDefault(),
-            MaxAutoscaleSmServiceAccountsChanged = 200 != organization.MaxAutoscaleSmServiceAccounts.GetValueOrDefault()
-        };
+        var organizationUpdate = new SecretsManagerSubscriptionUpdate(
+            organization, seatAdjustment: 5, maxAutoscaleSeats: 15, serviceAccountAdjustment: 100, maxAutoscaleServiceAccounts: 200);
 
         sutProvider.GetDependency<IOrganizationRepository>().GetByIdAsync(organizationId).Returns(organization);
 
         await sutProvider.Sut.UpdateSecretsManagerSubscription(organizationUpdate);
-        if (organizationUpdate.SmServiceAccountsAdjustment != 0)
-        {
-            await sutProvider.GetDependency<IPaymentService>().Received(1)
-                .AdjustSeatsAsync(organization, plan, organizationUpdate.SmSeatsExcludingBase);
-
-            // TODO: call ReferenceEventService - see AC-1481
-        }
 
         if (organizationUpdate.SmSeatsAdjustment != 0)
         {
             await sutProvider.GetDependency<IPaymentService>().Received(1)
                 .AdjustServiceAccountsAsync(organization, plan, organizationUpdate.SmServiceAccountsExcludingBase);
 
             // TODO: call ReferenceEventService - see AC-1481
-        }
 
-        if (organizationUpdate.SmSeatsAdjustment != 0)
-        {
             await sutProvider.GetDependency<IOrganizationService>().Received(1).ReplaceAndUpdateCacheAsync(
                 Arg.Is<Organization>(org => org.SmSeats == organizationUpdate.SmSeats));
         }
 
         if (organizationUpdate.SmServiceAccountsAdjustment != 0)
```

Comment:
Why do we have these `if` statements in here when the test data is static? Will the assertions ever change?